### PR TITLE
chat template file created

### PR DESCRIPTION
**Pull request by HARSHKALAL**

### CODE REVIEW

File: Chatify/chat/views.py [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+
+def register(request):
+    return render(request, "chat/register.html")
+
+
+def login(request):
+    return render(request, "chat/login.html")
+
 def chat(request):
     return render(request,"chat/chat.html")
```
Comment (Dhruvi-Makwana) [inline Chatify/chat/views.py]: add new line at end of file

File: Chatify/chat/urls.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+from django.urls import include, path
 
+app_name = "chat"
 urlpatterns = [
+    path("user_register/", register, name="user_register"),
+    path("login/", login, name="user_register"),
     path("chatui/",chat,name="chatui"),
-]
\ No newline at end of file
+]
```
Comment (Dhruvi-Makwana) [inline Chatify/chat/urls.py:10]: add new  line at end of file

File: Chatify/chat/urls.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-from django.urls import include, path
+from django.urls import path
+from .views import *
+
 
-app_name = 'chat'
 urlpatterns = [
-    path('user_register/', RegisterView.as_view(), name='user_register'),
+    path("chatui/",chat,name="chatui"),
 ]
```
Comment (Dhruvi-Makwana) [inline Chatify/chat/urls.py]: add new line at end of file, 
url name is not proper

File: Chatify/Chatify/settings/base.py [hunk 4]
```diff
@@ -61,9 +61,10 @@
         "USER": "postgres",
         "PASSWORD": "postgres",
         "HOST": "localhost",
-        "PORT": "5432",
+        "PORT": "",
```
Comment (yatishTrootech) [inline Chatify/Chatify/settings/base.py:64]: Port should not be blank

File: Chatify/chat/static/css/register.css [hunk 5]
```diff
@@ -112,20 +130,24 @@ form .field input[type="submit"]{
...
   color: #4158d0;
   text-decoration: none;
 }
+
 form .pass-link a:hover,
-form .signup-link a:hover{
+form .signup-link a:hover {
   text-decoration: underline;
-}
+}
```
Comment (yatishTrootech) [inline Chatify/chat/static/css/register.css:153]: add a line at the end of file

File: Chatify/chat/templates/chat/chat.html [hunk 6]
```diff
@@ -0,0 +1,228 @@
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:1]: Don;t use CDN

File: Chatify/chat/templates/chat/chat.html [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+        integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/malihu-custom-scrollbar-plugin/3.1.5/jquery.mCustomScrollbar.min.css">
+    <script type="text/javascript"
+        src="https://cdnjs.cloudflare.com/ajax/libs/malihu-custom-scrollbar-plugin/3.1.5/jquery.mCustomScrollbar.min.js"></script>
+</head>
+
+<body>
+    <div class="container-fluid h-100">
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:26]: Code should not be repeated use ANGULAR JS to bind users in array

File: Chatify/chat/templates/chat/chat.html [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+                                <li><i class="fas fa-user-circle"></i> View profile</li>
+                                <li><i class="fas fa-users"></i> Add to close friends</li>
+                                <li><i class="fas fa-plus"></i> Add to group</li>
+                                <li><i class="fas fa-ban"></i> Block</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="card-body msg_card_body">
+                        <div class="d-flex justify-content-start mb-4">
+                            <div class="img_cont_msg">
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:140]: Messages Should be from ANgular JS araay

File: Chatify/chat/templates/chat/chat.html [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
...yxZMDjtdjbjzqWn3Cq7OFkBVJBuj3c1FUYoIsaACVy2k5MWUgbR5olmEHBTsjxFgDWCXeKnO06B1YfBp+ylLWzBI7lt+GJ0UW7PQ+m4ZQx8l3YM7a2HH6D4grR6QCP6bPIQphs4C51W/wCVA0CWjc/d+0LnY59X43OA4AI3C4BpuDPepjmHwhqGqU3ON3X5KRU/jt7G2AIFRrYsJ07k0pbPFWWPJIHw3Ai2vKJKW7GwhgvO4Hx4lNWOH3V0YXE8513Wf3xZt+GMZScBIMyNHAAAH19EgfXa1rqj/hFo48kXtyoOtyt3Q3x3+pVT6RY2SKbfhHut+NcIEnl4w5Dn/e1vyu/zLSpmZYn7rKf6qQ+cSFznHcUQSoKjJVTOWcvXBK4JIsdFjikBp48CuRU46rCVG47j4IAIa9d50LTqbjquy5AwkPWi2NFCHLptRAGyVDUoBESCuXUjqECAbhdU3kEOYS1zTIO8KZzuIXBY06GEDRZMH0mYQBVaWu3uaJb3xqO66cse17Q5jg5u4gyO7v5KgGmeRUmAxtSg/My4/U3c4c+fNZp4F3RohnfaR6FQoyQTu9EXX7LSUDsjaVOqzMw94OrTwP5ddbaxwZRqO4NPqLesLLT5UbLVWQ4qg5jsr27gQZGjgCDz1WU6R/S49yTbF2uP5Uda4g0SGTBJyPksBgbiHgeCKb0iww/5jj3McrZQd9ju9L1mF4lckv8AY0a928HyWOSWp0oobhUd/wBoHuUtxXSs/wDLpBvNxJ9BAQsUvonk9T6aC/KyykTxPJQ0qwfUFGmQ6odYuGDe5x+nFUnFbVrVPiqGODeyPIJt/D+sW4sCJzNI7ogyFbHD9nH6n1dzTWNV/J6hSZlsP06fn0W3PABO6JOgsB+y6ul+3KmSi8zd0NHjr7FXxWziQuc0vsqONxPx1TxJHedFSazyTJVh6R1srWsHefRVtyun9G3qJbUfo5WLcrFEzbLDh1p2q2sSZSD1lDTW1iQGlDWWLECNvXe5YsQM6K2sWIGbRDFixAEWKQK2sQB1TWjqsWIGO+hf+O7/AOs+6b9MP8MfnBYsWR/sRqh+oRbM/wCDxv8A+b/2qJA7VbWLV4Mi7s2FzU1W1iGJHIVk6Af8a3+130WliQz1j90m6Tf4bf7x7OWLFLH+RZ037Eeb9Jf8XwSRYsU5dy7P+bOVixYokD//2Q=="
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:175]: don't Use base 64 instead use a link

File: Chatify/chat/templates/chat/login.html [hunk 10]
```diff
@@ -2,31 +2,34 @@
...
+        <label>Password</label>
+      </div>
 
-        <div class="field">
-          <input type="password" required>
-          <label>Password</label>
-        </div>
+      <div class="field">
+        <input type="submit" value="Login">
+      </div>
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/login.html:30]: Don;t submit form directly it should be submitted by login.JS AJAX

File: Chatify/chat/templates/chat/register.html [hunk 11]
```diff
@@ -1,44 +1,46 @@
-
 <!DOCTYPE html>
 {% load static %}
-<!-- Created By CodingLab - www.codinglabweb.com -->
+
 <html lang="en" dir="ltr">
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/register.html:4]: Add register.JS

File: Chatify/chat/templates/chat/chat.html [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<!------ Include the above in your HEAD tag ---------->
+
+<!DOCTYPE html>
+{% load static %}
+
+<html>
+
+<head>
+    <title>Chat</title>
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:12]: Add a http_utils.JS
Add ajax code there

File: Chatify/chat/templates/chat/chat.html [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<!------ Include the above in your HEAD tag ---------->
+
+<!DOCTYPE html>
+{% load static %}
+
+<html>
+
+<head>
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:11]: Every JS which is linked with html should be a Angular JS controller file

File: Chatify/chat/templates/chat/chat.html [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+    <title>Chat</title>
+    <link rel="stylesheet" href="{% static 'css/chat.css' %}">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+        integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+        integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <link rel="stylesheet" type="text/css"
```
Comment (yatishTrootech) [inline Chatify/chat/templates/chat/chat.html:19]: No HTML will have JS code

**Review by yatishTrootech — CHANGES_REQUESTED** [review]

**Pull request closed without merging**